prep-css-box-model/styles.css

```diff
@@ -0,0 +1,18 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  margin: 20px;
+  padding: 10px;
+
+  p.double {
+    width: 3px;
+    border-style: double;
+    color: darkgray;
+  }
```

Comment:
Hi Justin, this rule set is extra, you can delete it. We just need border-width, border-style, and border-color properties added to the `button` rule set

Reply:
Hi Tim, 
Completed! Thanks!